docs/getting-started.md

```diff
@@ -37,7 +37,7 @@
 
 ### 2. Install [Cachix](https://cachix.org)
 
-Recommended, speeds up the installation by providing binaries.
+Speeds up the installation by providing binaries.
```

Comment:
Even this wording suggests that it's still optional. Any ideas for better description? As it's not only speeds up the installation, but is actually required.